apps/src/studio/studio.js

```diff
@@ -657,6 +646,8 @@ var cancelQueuedMovements = function (index, yAxis) {
         cmd.opts.queuedDistance = 0;
       } else if (!yAxis && (dir === Direction.EAST || dir === Direction.WEST)) {
         cmd.opts.queuedDistance = 0;
+      } else if (!CardinalDirections.includes(dir)) {
```

Comment:
It works for me in IE, I was under the impression that https://github.com/code-dot-org/code-dot-org/pull/8634 gave us access to all the ES6 features in apps and code-studio. Is that not how it works?